pages/dashboard/dashboard.css

```diff
@@ -0,0 +1,50 @@
+.cards {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(18.75rem, 30%));
+  grid-gap: 1.25rem;
+  margin-top: 3.75rem;
+}
+
+.card {
+  background-color: var(--white);
+  border-radius: 5px;
+  padding: 0.625rem;
+  box-shadow: 0 0 10px #0000001a;
+  cursor: pointer;
+}
+
+img {
+  width: 100%;
+  height: 200px;
+  object-fit: cover;
+  border-radius: 5px;
+}
+
+h2 {
+  padding: 1.25rem 0;
+}
+
+.tags {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 0.5rem;
+  padding-top: 1rem;
+}
+
+button {
+  background-color: var(--orange);
+  color: var(--white);
+  padding: 0.313rem 0.625rem;
+  border-radius: 5px;
+  font-size: 0.875rem;
+  font-weight: 500;
+  border: none;
+  cursor: inherit;
+}
+
+/* max-width: 425px */
+@media(max-width: 26.563rem) {
+  .cards {
+    grid-template-columns: 1fr;
```

Comment:
postoji jedan trik s gridom u kojem ne moras uopce koristiti media queries da ti bude responzivno. 

https://bitsofco.de/how-the-minmax-function-works/